content/en/tracing/trace_collection/compatibility/dotnet-core.md

```diff
@@ -49,12 +49,60 @@ The .NET Tracer supports automatic instrumentation on the following architecture
 
 | Processor architectures                   | Support level         | Package version                        |
 | ------------------------------------------|-----------------------|----------------------------------------|
-| Windows x86 (`win-x86`)                   | [GA](#support-ga)     | latest                                 |
 | Windows x64 (`win-x64`)                   | [GA](#support-ga)     | latest                                 |
+| Windows x86 (`win-x86`)                   | [EOL](#support-eol)   | < 3.0.0 (e.g. 2.48.0)                  |
 | Linux x64 (`linux-x64`)                   | [GA](#support-ga)     | latest                                 |
 | Alpine Linux x64 (`linux-musl-x64`)       | [GA](#support-ga)     | latest                                 |
 | Linux ARM64 (`linux-arm64`)               | [GA](#support-ga)     | .NET 5+ only, added in version 1.27.0  |
 
+Note that running x86 applications on Windows x64 is supported.
+
+## Supported operating systems
+
+The .NET Tracer supports automatic instrumentation on Windows and Linux operating systems. It supports macOS for CI Test Visibility only.
+
+### Windows
+
+| Operating System             | Version     | Support level         | Package version                        |
+| -----------------------------|-------------|-----------------------|----------------------------------------|
+| Windows Server (x64)         | 2012+       | [GA](#support-ga)     | latest                                 |
+| Windows Client (x64)         | 8.1+        | [GA](#support-ga)     | latest                                 |
```

Comment:
Just curious, is the term Windows Client widely known? As a .NET expert I get what you're saying, but I wonder if it's clear to other readers

Reply:
I 100% based it on the .NET wording 😄 https://github.com/dotnet/core/blob/026c5d2a974027e32eb0faeef3b9401923a3bd7f/release-notes/3.1/3.1-supported-os.md?plain=1#L17

I guess it makes sense as a counter-point to "Server", but I probably would have expected to see "Desktop" personally? 🤔

Reply:
Strange. Welp, I won't argue with the .NET's wording 😅